README.md

```diff
@@ -26,4 +28,82 @@ An outcome accuracy of at least 65% for startup outcome classification.
 
 Inbound traffic from 100 existing venture capital firms.
 
+## Planned Work
 
+### Theme 1: A dataset that is effective in representing every facet of the venture capital ecosystem 
+
+- E1 - Aggregate general startup data during time period
+	- S1 : Cleaned data on general information for each startup 
+	- S2 : New features based on funding frequencies, amounts, target market.
+- E2 - Venture round trends and characteristics
+	- S3 : Cleaned data on venture rounds that were raised by startups during this period.
+	- S4 : New features based on the type of venture round that was raised - seed, growth etc.
+	- S5 : Venture type and amount trends across time period.
+- E3 - Data on venture capital investors who invested during time period
+	- S6 : Cleaned data on venture investors during time period.
+	- S7 : Scoring of venture investors based on frequency, amount of fundings.
+	- S8 : Social media and online mention frequency and influence of venture capital investors.
+- E4 - Mergers and acquisitions that happened to companies in question
+	- S9 : Cleaned data on mergers and acquisitions during this period.
+	- S10 : New features based on how accretive acquisition was.
+	- S11 : Success metric and scoring to gauge venture outcome.
+- E5 - Macro environment and trends during time period
+	- S12 : Acquired external data on macro trends during time period.
+	- S13 : Cleaned data on external macro trends.
+	- S14 : New features based on how macro trends compare across different time periods.
+
+### Theme 2: Machine learning models that help predict different venture outcomes
+
+- E6 - Predict total funding amount of startup
+	- S15 : Grouped, aggregated, and in-depth EDA across all collected data.
+	- S16 : New features from combined datasets.
+	- S19 : Base regression model for funding amounts.
+	- S20 : Model selection and model tuning for regression model on funding amounts.
+- E7 - Predict startup/venture outcome - whether is still operating, acquired, went public
+	- S17 : Base logistic regression model for multilabel classification.
+	- S18 : Model selection and model tuning for multilabel classification.
```

Comment:
Does "model selection" include investigating more complex models than the above mentioned logistic regression?

Reply:
Could add an icebox on alternatives outside of logistic and regression models.